modules/polling/api/__tests__/sortPollList.ts

```diff
@@ -0,0 +1,70 @@
+/*
+
+SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+
+*/
+
+import { mockPollList } from '../mocks/pollList';
+import { sortPollsBy } from 'modules/polling/helpers/sortPolls';
+import { PollListItem } from 'modules/polling/types';
+import { PollOrderByEnum } from 'modules/polling/polling.constants';
+
+const mockPollIds = [967, 982, 980, 957, 956, 952];
+
+const shufflePolls = (pollArray: PollListItem[]): PollListItem[] => {
+  for (let i = pollArray.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    const temp = pollArray[i];
+    pollArray[i] = pollArray[j];
+    pollArray[j] = temp;
+  }
+  return pollArray;
+};
+
+describe('Sort polls', () => {
```

Comment:
should add `.spec` to this file name to not get it confused with actual code files